field/src/batch_inverse.rs

```diff
@@ -12,8 +15,26 @@ use crate::field::Field;
 /// compute WIDTH separate cumulative product arrays that only meet at the end.
 ///
 /// # Panics
-/// Might panic if asserts or unwraps uncover a bug.
+/// This will panic if any of the inputs is zero.
+#[instrument(level = "debug", skip_all)]
 pub fn batch_multiplicative_inverse<F: Field>(x: &[F]) -> Vec<F> {
+    // How many elements to invert in one thread.
+    const CHUNK_SIZE: usize = 1024;
+
+    let n = x.len();
+    let mut result = vec![F::zero(); n];
```

Comment:
Might as well use our fancy new F::zero_vec for this :)

Reply:
Good idea, done